mitodl/hugo-course-publisher
# other_information_text

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/135

#### What's this PR do?
This PR replaces the call for `.Params.course_description` piped to `safeHTML` in the course home page template with a single call to `{{ .Content }}` because this and the new `other_information_text` field are now brought in as markdown content on the course home page `_index.md` file.

#### How should this be manually tested?
Run the site, verify that it builds without error and you can see the previously missing "Course Format" section here:

http://localhost:3000/courses/14-01sc-principles-of-microeconomics-fall-2011/

#### Screenshots (if appropriate)
![Screen Shot 2020-07-09 at 5 47 11 PM](https://user-images.githubusercontent.com/12089658/87094367-c80e5500-c20c-11ea-8dbb-737a44de9f10.png)
![Screen Shot 2020-07-09 at 5 50 36 PM](https://user-images.githubusercontent.com/12089658/87094377-ca70af00-c20c-11ea-8ea8-725a954a8e34.png)
